platform/ATCmdParser.h

```diff
@@ -282,7 +300,7 @@ class ATCmdParser : private NonCopyable<ATCmdParser> {
      */
     int scanf(const char *format, ...) MBED_SCANF_METHOD(1, 2);
 
-    int vscanf(const char *format, std::va_list args);
+    int vscanf(const char *response, std::va_list args);
```

Comment:
This could be reverted back fully, to be in line with `scanf`

Reply:
Done